packages/integrations/image/README.md

```diff
@@ -149,10 +149,10 @@ Source for the original image file.
 
 For images located in your project's `src`: use the file path relative to the `src` directory. (e.g. `src="../assets/source-pic.png"`)
 
-For images located in your `public` directory: use the URL path relative to the `public` directory. (e.g. `src="/images/public-image.jpg"`)
-
 For remote images, provide the full URL. (e.g. `src="https://astro.build/assets/blog/astro-1-release-update.avif"`)
 
+For images located in your `public` directory: use the URL path relative to the `public` directory. (e.g. `src="/images/public-image.jpg"`). These work like remote images.
```

Comment:
I guess the reasoning for moving this here is so “like remote images” follows an explanation of remote images. I think it’s not ideal to split `src` and `public` though. How about we show remote, local, public in that order?

Style note: we usually show directories with a trailing slash so we maybe want to update these to be `src/`, `public/` etc?

Reply:
done!